accurest-core/src/test/groovy/io/codearte/accurest/builder/MockMvcSpockMethodBuilderSpec.groovy

```diff
@@ -836,4 +838,65 @@ World.''')
 World.'''""")
 	}
 
+	@Issue('180')
+	def "should generate proper test code when having multipart parameters"(){
+		given:
+			GroovyDsl contractDsl = GroovyDsl.make {
+				request {
+					method "PUT"
+					url "/multipart"
+					headers {
+						header('content-type', 'multipart/form-data;boundary=AaB03x')
+					}
+					multipart(
+							formParameter: value(client(regex('.+')), server('"formParameterValue"')),
```

Comment:
I can't find it but are you checking if the server value is matching the regex in on the client side?